vendor/bat-native-ads/src/bat/ads/internal/ads/serving/eligible_ads/exclusion_rules/split_test_exclusion_rule_unittest.cc

```diff
@@ -20,9 +21,9 @@ constexpr char kCreativeSetId[] = "654f10df-fbc4-4a92-8d43-2edf73734a60";
 
 scoped_refptr<base::FieldTrial> CreateFieldTrial(
     const std::string& trial_name) {
-  return base::FieldTrialList::FactoryGetFieldTrial(
-      trial_name, 100, "default",
-      base::FieldTrialList::GetEntropyProviderForSessionRandomization());
+  base::MockEntropyProvider entropy_provider(0.9);
```

Comment:
Rather than `0.9` should we use `RandDouble()` to match the previous provider?

Reply:
From what I can tell we just care that a FieldTrial exists. Upstream seems to use specific values in their tests, but I am not sure if it would make a difference for these tests. WDYT, @cdesouza-chromium?